Composer/packages/ui-plugins/virtual-assistant-creation/src/shared/util.ts

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
What happens if the fetch call fail?

Reply:
I added a console log if there is an error on the fetch, are there any additional recommendations? Note the long term solution will probably not have this GitHub API call as we determine the long term hosting plan for these personality .qna files.

Reply:
No matter how reliable the backend, the call still can fail, like a slow internet connection can cause a timeout or etc.